packages/dashboard/src/components/appbar.tsx

```diff
@@ -229,15 +229,21 @@ export const AppBar = React.memo(({ extraToolbarItems }: AppBarProps): React.Rea
       }
       if (!schedule) {
         await Promise.all(
-          taskRequests.map((request) =>
-            rmf.tasksApi.postDispatchTaskTasksDispatchTaskPost({
+          taskRequests.map((request) => {
+            console.log(`submitTask: ${request}`);
```

Comment:
These `console.log` could be replaced with `debug`.

Reply:
I'll clean them up as debug messages after the demo then